tests/conftest.py

```diff
@@ -100,3 +100,18 @@ def dummy_config(test_seed, dummy_split, dummy_dataset_name, dummy_model_name):
         "n_samples": 50,
         "random_state": test_seed,
     }
+
+
+@pytest.fixture
+def dummy_validation_scores(rng, test_n_samples):
+    return np.sort(rng.uniform(0, 1, test_n_samples)) * 100
+
+
+@pytest.fixture
+def dummy_perfect_metric_scores(rng, test_n_samples):
+    return np.sort(rng.uniform(0, 1, test_n_samples)) * 100
```

Comment:
```suggestion
def dummy_perfect_metric_scores(dummy_validation_scores):
    return dummy_validation_scores
```

I think this might be clearer (and less chance that we change one and forget to update the other at some point)?